plugins/pulp_docker/plugins/registry.py

```diff
@@ -368,8 +368,39 @@ def get_manifest(self, reference):
         :return:          A 2-tuple of the digest and the manifest, both basestrings
         :rtype:           tuple
         """
+        manifests = []
+        request_headers = {}
+        content_type_header = 'content-type'
+        schema1 = 'application/vnd.docker.distribution.manifest.v1+json'
+        schema2 = 'application/vnd.docker.distribution.manifest.v2+json'
         path = self.MANIFEST_PATH.format(name=self.name, reference=reference)
-        headers, manifest = self._get_path(path)
+        # set the headers for first request
+        request_headers['Accept'] = schema2
+        headers, manifest = self._get_path(path, headers=request_headers)
```

Comment:
https://github.com/pulp/pulp_docker/pull/187/files#diff-99c0d67fc4b80f3d190c6b6af1c453b1R447
i could rename `headers' to `request_headers' if that would make it better